docs/user_guide/i18n.rst

```diff
@@ -0,0 +1,61 @@
+Internationalization
+====================
+
```

Comment:
Perhaps we should split this into two sections. One for theme built-in strings, and another for user configurable strings.

Reply:
The first paragraph is built-in strings, the rest is user-configurable strings.